src/vm-memory/Cargo.toml

```diff
@@ -7,6 +7,6 @@ license = "Apache-2.0"
 
 [dependencies]
 libc = ">=0.2.80"
-vm-memory-upstream = { package = "vm-memory",  version = "0.6.0", features = ["backend-mmap", "backend-bitmap"] }
+vm-memory-upstream = { package = "vm-memory",  version = "0.7.0", features = ["backend-mmap", "backend-bitmap"] }
```

Comment:
Not necessarily related, because `zerioize` comes with no dependencies, but I thought it would be a good idea to unify versions for `vm-memory` - since `io-uring` crate is using "0.7.0". WDYT?

Reply:
That's fine by me, but should probably be in a separate commit

Reply:
Actually, I'd rather have this change merged: https://github.com/firecracker-microvm/firecracker/pull/2843 and update the vm-memory version after 1.0.0 release